cookbooks/fb_init_sample/recipes/default.rb

```diff
@@ -95,7 +95,9 @@
   include_recipe 'fb_hdparm'
   include_recipe 'fb_sdparm'
   include_recipe 'fb_nscd'
-  include_recipe 'fb_hddtemp'
+  if fedora_derived?
+    include_recipe 'fb_hddtemp'
+  end
```

Comment:
@michel-slm should we try to get this resurrected in Debian? looks like it was [retired a while ago](https://packages.qa.debian.org/h/hddtemp.html)